kas/README.md

```diff
@@ -0,0 +1,33 @@
+# Building
+
+To start a full build, e.g. for the Kirkstone release, run the following command:
+
+    kas build kas/leda-kirkstone.yaml
+
+    kas build -c cleanall kas/leda-kirkstone.yaml k3s
+
+kas will then check out all necessary dependencies as specified in the project configuration file. After that, it will start the build for the machine, distro and target device.
+
+# Running
+
+Running with user-level networking, in case the host does not have TAP/TUN set up properly. If you have TAP/TUN set up, remove the `slirp` keyword. Also enable the OVMF bios for multiboot. If you have kvm enabled on your host, add the `kvm` keyword:
+
+    kas shell kas/leda-kirkstone.yaml -c 'runqemu slirp nographic ovmf'
+
+Boot directly into a specific partition:
+
+    kas shell kas/leda-kirkstone.yaml -c 'runqemu slirp nographic ovmf sdv-image-all'
+    kas shell kas/leda-kirkstone.yaml -c 'runqemu slirp nographic ovmf sdv-image-full'
+    kas shell kas/leda-kirkstone.yaml -c 'runqemu slirp nographic ovmf sdv-image-minimal'
+    kas shell kas/leda-kirkstone.yaml -c 'runqemu slirp nographic ovmf sdv-image-rescue'
+
```

Comment:
By default we should boot to sdv-image-all, otherwise the fstab mounts within each of the mentioned partitions (full, minimal, rescue) will not match the disk setup in qemu and you'll get error messages during boot and the mount points for /data will be missing.